src/main/services/torrent-client.ts

```diff
@@ -22,6 +22,7 @@ enum TorrentState {
   Downloading = 3,
   Finished = 4,
   Seeding = 5,
+  DebridCaching = 6,
```

Comment:
Make sure not to mix this with `libtorrent` statuses
If you want to create such thing as a virtual status, use a constant that is not being used by libtorrent
https://libtorrent.org/reference-Torrent_Status.html

Reply:
Done ✅